test/utils_test.rb

```diff
@@ -0,0 +1,118 @@
+# typed: false
+# frozen_string_literal: true
+
+require_relative "./test_helper"
+
+module ShopifyAPITest
+  class UtilsTest < Test::Unit::TestCase
+    extend T::Sig
+
+    def setup
+      super
+
+      test_session = ShopifyAPI::Auth::Session.new(
+        id: "id",
+        shop: "test-shop.myshopify.io",
+        access_token: "this_is_a_test_token"
+      )
+      ShopifyAPI::Context.activate_session(test_session)
+      modify_context(api_version: "2022-04")
```

Comment:
Complete nit, but we probably want to do this before activating the session in case we ever change it to reset the session.